<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="169" started-at="2019-05-03T08:18:30Z" finished-at="2019-05-03T08:18:30Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="169" started-at="2019-05-03T08:18:30Z" finished-at="2019-05-03T08:18:30Z">
      <class name="com.matryxsoft.website.matryxsoft.helloworld">
        <test-method status="PASS" signature="MatryxsoftWebsite()[pri:0, instance:com.matryxsoft.website.matryxsoft.helloworld@2133c8f8]" name="MatryxsoftWebsite" duration-ms="7" started-at="2019-05-03T13:48:30Z" finished-at="2019-05-03T13:48:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MatryxsoftWebsite -->
        <test-method status="FAIL" signature="MatryxsoftWebsite2()[pri:0, instance:com.matryxsoft.website.matryxsoft.helloworld@2133c8f8]" name="MatryxsoftWebsite2" duration-ms="1" started-at="2019-05-03T13:48:30Z" finished-at="2019-05-03T13:48:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.matryxsoft.website.matryxsoft.helloworld.MatryxsoftWebsite2(helloworld.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:73)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:94)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MatryxsoftWebsite2 -->
      </class> <!-- com.matryxsoft.website.matryxsoft.helloworld -->
      <class name="com.matryxsoft.website.matryxsoft.reRunFailedTestNG">
        <test-method status="PASS" signature="reRunFailedTests()[pri:0, instance:com.matryxsoft.website.matryxsoft.reRunFailedTestNG@5594a1b5]" name="reRunFailedTests" duration-ms="126" started-at="2019-05-03T13:48:30Z" finished-at="2019-05-03T13:48:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reRunFailedTests -->
      </class> <!-- com.matryxsoft.website.matryxsoft.reRunFailedTestNG -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
